### DISCUSSION TITLE
Overexposed images for small ROI

### DISCUSSION THREAD

**Discussion by Karl-Philippe** [PlusToolkit/PlusLib#996] [Q&A] [answered]
Hello, 

I use a PLUS server (Microsoft Media Foundation compatible imaging device) to acquire ultrasound images with a frame grabber. However, I am experiencing a problem as the smaller the region of interest (ClipRectangleOrigin and ClipRectangleSize), the more the images are overexposed (high brightness). When no cropping is applied using the ClipRectangleOrigin/Size parameters, the images captured in Slicer look exactly the same as they do on the ultrasound system monitor. Is this a known problem and what would be the solution to not having overexposed images?

Thank you,

My hardware is:
- Ultrasound system Acuson S3000
- Frame grabber AV.io HD epiphan video
- Window computer

Here is my device configuration:

    <Device
      Id="VideoDevice" 
      Type="MmfVideo" 
      VideoFormat="YUY2"
      FrameSize="1280 720"
      CaptureDeviceId="0" >
      <DataSources>
        <DataSource Type="Video" Id="Video" PortUsImageOrientation="UN" ImageType="BRIGHTNESS" 
          ClipRectangleOrigin = "405 38" 
          ClipRectangleSize = "460 450" />
      </DataSources>      
      <OutputChannels>
        <OutputChannel Id="VideoStream" VideoDataSourceId="Video" />
      </OutputChannels>
    </Device>

**Comment by Sunderlandkyl** [comment, marked as answer]
Probably the window-level set in Slicer is not the same with the cropped ultrasound.

You can check and adjust the exact window level values in the Volumes module (in the display section):
![image](https://user-images.githubusercontent.com/9222709/196451278-2d4f813a-7b59-4fe3-9d7a-8b58c26fa74e.png)

You can also use the window-level mouse mode to adjust the contrast by clicking and dragging on the image:
![image](https://user-images.githubusercontent.com/9222709/196451048-26fcace8-64a7-442d-aaf1-06d7131fb374.png)
